src/styles/quest.less

```
@@ -37,15 +37,22 @@
     line-height: 1rem;
   }
   
-  #questLines {
-    .card-header {
+  .card-header {
       position: relative;
+	  margin-top : 1px;
       
       .info {
         position: absolute;
         top: 0px;
         right: 5px;
       }
     }
-  }
+	
+	table {
+		margin-bottom : 4px;
+	}
 }
+
+.questDisplayBlock:last-child table {
+	margin-bottom : 0px;
+}
```

I believe this should also work with less, without having to do it separately: 
```suggestion
    }
    &:last-child table {
        margin-bottom : 0px;
    }
}
```